src/check_update.c

```diff
@@ -63,13 +65,24 @@ enum swupd_code check_update()
 		ret = SWUPD_SERVER_CONNECTION_ERROR;
 	}
 
+	if (globals.verbose) {
+		latest_format = get_latest_format(server_version);
+		get_current_format(&current_format);
+	}
+
 	if (current_version > 0) {
-		info("Current OS version: %d\n", current_version);
+		info("Current OS version: %d", current_version);
+		info_verbose(" (format %s)", (current_format == NULL) ? "unknown" : current_format);
+		info("\n");
 	}
 	if (server_version > 0) {
-		info("Latest server version: %d\n", server_version);
+		info("Latest server version: %d", server_version);
+		info_verbose(" (format %d)", latest_format);
```

Comment:
do the same you did for current format to print unknown if latest_format < 0

Reply:
Done